src/main/resources/templates/authentication/register.html

```diff
@@ -41,7 +44,8 @@
                      aria-labelledby="nav-student-tab">
                     <br>
                     <h3>Student Registration</h3>
-                    <form th:object="${user}" method="post" action="/register/student">
+                    <form id="student-form" th:object="${user}" method="post" action="/register/student">
+                        <div id = "student-error"></div>
```

Comment:
@ThePupoMaster please format this into a warning/danger bootstrap alert

Reply:
https://v5.getbootstrap.com/docs/5.0/components/alerts/